{"name": "test_list_management_users", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\jothi\\\\Documents\\\\api-testing-repo\\\\tests\\\\test_data.json'", "trace": "cls = <class 'management_user.TestManagementUserAuth'>\n\n    @classmethod\n    def setUpClass(cls):\n        cls.api = ManagementUserAPIHandler(base_url=\"http://localhost:8000\")\n        json_path = os.path.join(os.path.dirname(__file__), \"test_data.json\")\n    \n>       with open(json_path, \"r\") as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\jothi\\\\Documents\\\\api-testing-repo\\\\tests\\\\test_data.json'\n\nmanagement_user.py:17: FileNotFoundError"}, "start": 1754042418838, "stop": 1754042418838, "uuid": "a1818ad6-e778-455a-a870-13ce819fab4a", "historyId": "e74131c91c3bf1b4144873b5425b8b25", "testCaseId": "e74131c91c3bf1b4144873b5425b8b25", "fullName": "tests.management_user.TestManagementUserAuth#test_list_management_users", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "management_user"}, {"name": "subSuite", "value": "TestManagementUserAuth"}, {"name": "host", "value": "mirrajo"}, {"name": "thread", "value": "5044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.management_user"}], "titlePath": ["tests", "management_user.py", "TestManagementUserAuth"]}